wandb/sdk/launch/registry/google_artifact_registry.py

```diff
@@ -211,7 +211,7 @@ async def check_image_exists(self, image_uri: str) -> bool:
             for image in await list_images(request={"parent": parent}):
                 if tag in image.tags:
                     return True
-        except google.api_core.exceptions.NotFound as e:
+        except google.api_core.exceptions.NotFound as e:  # type: ignore[attr-defined]
```

Comment:
what's the issue here?

Reply:
missing Google types

Reply:
I’ll look on type shad or something, but cursory search didn’t yield much